uint32                      id
string                      job_id
mac_ros_bridge/Position     destination
float64                     deadline
string[]                    items
